scripts/monitor-plugin-nuts.js

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env node
+/**
+ * Script to monitor previously launched just-nuts
+ *
+ * args:
+ *   string representation of a jsoan array that contains the data for each pipeline/workflow to monitor
+ *
+ */
+
+const got = require('got');
+const { isArray } = require('@salesforce/ts-types');
+const { sleep, Duration } = require('@salesforce/kit');
+
+const notCompleteStatus = ['running', 'on_hold'];
+
+// circleci
+const circleciBaseUrl = 'https://circleci.com/api/v2/';
+
+const orgSlug = (org, repo) => {
+  return `github/${org}`;
+};
+const projectSlug = (org, repo) => {
+  return `${orgSlug(org)}/${repo}`;
+};
+
+class MonitorPluginNuts {
+  pipelineData = {};
+  isComplete = false;
+  status = 'initial state';
+  justNuts = {};
+  constructor(pipelineData) {
+    this.pipelineData = pipelineData;
+  }
+
+  getWorkflowUrl() {
+    return `${circleciBaseUrl}pipeline/${this.pipelineData.id}/workflow`;
+  }
+
+  getCircleCiUrl() {
+    return `https://app.circleci.com/pipelines/${this.justNuts.project_slug}/${this.justNuts.pipeline_number}/workflows/${this.justNuts.id}`;
+  }
+
+  async circle(url, options = {}) {
+    const defaultOptions = {
+      method: 'GET',
+      headers: { 'Circle-Token': process.env.CIRCLE_TOKEN, responseType: 'json', resolveBodyOnly: true },
+    };
+    options = Object.assign(defaultOptions, { ...options });
+    try {
+      return JSON.parse((await got(url, options)).body);
+    } catch (error) {
+      console.log(error);
+      throw new Error(error && error.response?.body ? error.response.body : error);
+    }
+  }
+
+  async checkWorkflowState() {
```

Comment:
Checks workflow state. Right now this is only interested in workflows with the name "just-nuts"